core/src/main/java/nl/nn/adapterframework/pipes/LogContextPipe.java

```diff
@@ -56,12 +57,15 @@ public PipeRunResult doPipe(Message message, PipeLineSession session) throws Pip
 		if (!getParameterList().isEmpty()) {
 			Map<String,String> values = new LinkedHashMap<>();
 			try {
+				if (!message.isRepeatable()) {
+					message.preserve();
+				}
 				ParameterValueList pvl = getParameterList().getValues(message, session);
 				for(ParameterValue pv : pvl) {
 					values.put(pv.getName(), pv.asStringValue());
 				}
-			} catch (ParameterException e) {
-				throw new PipeRunException(this, "exception extracting parameters", e);
+			} catch (Exception e) {
+				log.debug("Exception getting parameter values. Ignoring.", e);
```

Comment:
Could this be changed to a `warn` statement?

If you see this a lot in production, something might be wrongly/sub-optimal configured..

Reply:
I was doubting because some messages we did make warnings, turned out to be excessive without pointing to any kind of misconfiguration.
But in this case you're probably right.